packages/components/src/utils/config-values.js

```diff
@@ -5,14 +5,16 @@ import { space } from './space';
 import { COLORS } from './colors-values';
 
 const CONTROL_HEIGHT = '36px';
-const CONTROL_PADDING_X = '12px';
```

Comment:
We should soon make a decision about how we want to define our foundational variables — number, string, or function (`space()`)? I am kind of leaning toward numbers because it's more versatile. For example we can make most calculations at compile time rather than rely on all the `calc()`s at runtime.

Reply:
Absolutely.

A few considerations that come to mind (more of a brain dump):

- Is this something that should come directly from theming?
- Using calc and CSS variables would enable a different style of coding — we would be able to change the value of a variable, instead of re-writing a new formula
-I agree that dealing with numbers is more practical. We could still have an underlying grid system, but we should consider exposing variables/tokens expressing numbers